openpype/hosts/resolve/api/plugin.py

```diff
@@ -291,17 +291,17 @@ class ClipLoader:
     active_bin = None
     data = dict()
 
-    def __init__(self, cls, context, path, **options):
+    def __init__(self, loader_obj, context, path, **options):
         """ Initialize object
 
         Arguments:
-            cls (openpype.pipeline.load.LoaderPlugin): plugin object
+            loader_obj (openpype.pipeline.load.LoaderPlugin): plugin object
             context (dict): loader plugin context
             options (dict)[optional]: possible keys:
                 projectBinPath: "path/to/binItem"
 
         """
-        self.__dict__.update(cls.__dict__)
+        self.__dict__.update(loader_obj.__dict__)
```

Comment:
What's the reason we need to do this? Instead of dedicated passing the actual arguments that we expect to exist? It's quite hard to grasp what data we're now expecting on the loader obj?

Reply:
Agree that this needs some refactor. This is old code of mine created 3 years ago with the knowledge of those days 🥲 

But since it is not the themem of this PR lets leave it as it is and move on. This refactor should be done separatelly.